ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.html

```diff
@@ -105,7 +107,7 @@ <h2 class="title" translate>App.HomePage.StayInformedSection.Title</h2>
     </section>
 
     <section class="join-cmi">
-        <img class="join-cmi-logo" lazy-resource src="assets/images/hands.png" [alt]="'Common.Alts.JoinCMI' | translate">
+        <img class="join-cmi-logo" lazy-resource src="assets/images/hands.png" [alt]="'App.HomePage.ImageAlt' | translate">
```

Comment:
'App.HomePage.ImageAlt' - not existing path, something missed there

Reply:
thanks, fixed!